css/index.css

```
@@ -1,3 +1,251 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 900px;
+	margin: auto;
+}
+
+header nav {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin-top: 25px;
+	margin-bottom: 25px;
+}
+
+header nav a {
+	text-decoration: none;
+}
+
+.hero {
+	margin-top: 40px;
+	align-items: center;
+	text-align: center;
+	display: flex;
+	justify-content: center;
+}
+
+.hero-text h1 {
+	 font-size: 75px;
+	 margin-right: 20px;	
+}
+
+.hero-text button{
+	font-size: 20px;
+    padding: 5px 40px 5px 40px;	
+}
+
+.hero img {
+	margin-left: 20px;
+}
+
+.home-top-content {
+	display: flex;
+	border-top: 1px solid black;
+	padding-top: 30px;
+	margin-top: 30px;
+	margin-bottom: 30px
+}
+
+
+.home-text-content {
+	width: 50%
+}
+
+.home-bottom-content {
+	margin-top: 30px;
+	border-bottom: 1px solid black;
+	display: flex;
+	justify-content: space-between;
+	padding-bottom: 30px;
+}
+
+.bottom-text-content {
+	width: 95%;
+	
+}
+
+.contact {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin-top: 30px;
+	height: 130px;
+}
+
+footer {
+	display: flex;
+	justify-content: center;
+	margin-top: 80px;
+	margin-bottom: 40px;
+}
+
+/* ------------ Tablet Media Query ------------ */
+
+@media (max-width: 800px){
+	
+	header {
+		display: flex;
+		flex-direction: column;
+		justify-content: space-around;
+		margin-top: 30px;
+		margin-left: 12px;
+		margin-right: 45px;
+		font-size: 16px;
+	}
```

```suggestion
	align-items: stretch;
	}
```
My proposed solution to your nav bar messes up your tablet view: this will fix that